### ISSUE TITLE
Signin form doesn't submit using return key

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/mobileweb2022#10]
If using a desktop machine, the form doesn't submit with the return key.

Although the site is not intended for desktop, this should be supported, as disabling a standard form mechanism indicates poor accessibility.
